CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.2.0-rc.1 December 2, 2021
+
+## Features ✨ and improvements 🏁
+* Cancellable queryRenderFeatures ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```

Comment:
And move to bugs as cancelable QRF were already introduced before
```suggestion
* Fix suggestion message to replace deprecated `queryRenderFeatures` overloaded methods. ([#878](https://github.com/mapbox/mapbox-maps-android/pull/878))
```

Reply:
Is it more a bugfix, nor an improvement / feature?

Reply:
@kiryldz probably not a `suggestion message` but `replaceWith` template or similar name